_drafts/fwup-signing.md

```diff
@@ -0,0 +1,642 @@
+---
+title: "Authenticate firmware builds with code signing"
+description: WIP
+author: francois
+---
+
+<!-- excerpt start -->
+<!-- excerpt end -->
+
+{% include newsletter.html %}
+{% include toc.html %}
+
+## Firmware Signing Explained
+
+### What is code signing
```

Comment:
You can probably remove this specific header

Reply:
:+1: